### PR TITLE
LandingPage

### PR DESCRIPTION
**Pull request by wfish-ghoti**
Assignment 6.1

### CODE REVIEW

**Comment by rbwatson** [comment]
rebased PR to include current `main` branch

**Review by rbwatson — CHANGES_REQUESTED** [review]
Please try to use more markdown and less HTML in the topic source code.

**Comment by wfish-ghoti** [comment]
> Please try to use more markdown and less HTML in the topic source code.

Hi Bob, I'm sorry. I thought you said that HTML was okay to use for this assignment. 
image

Would you like me to redo in md only?

**Comment by rbwatson** [comment]
Squashed commits and rebased to current `main` branch.

**Comment by rbwatson** [comment]
Fixed some links that had changed since PR was made.
Squashed all commits & rebased.

File: docs/Overview_wuf.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+## ![alt text](images/checkmark-png-25954_50.png)GET IT DONE
+
+Feel the satisfaction of checking items off your list.
+
+### [Learn More][def]
+
+## ![alt text](images/checkmark-png-25954_50.png)GET STARTED
+
+Our [Quickstart Guide](Quickstart_wuf.md) will get you up and running in **15 minutes**! Plus, we've got tutorials to help you out with whatever you need.
```
Comment (rbwatson) [inline docs/Overview_wuf.md:25]: Changed link to markdown link.

File: docs/contactus.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Contact Us
+
+* Chat with us: **opens-fake-chat-window**
```
Comment (rbwatson) [inline docs/contactus.md:3]: Removed fake link to pass CI test

File: docs/index.md [hunk 3]
```diff
@@ -25,6 +25,7 @@ Here are some more overview pages that explain the To-Do Service and how to get
 * [Sophie Yang's overview](overview_sy.md)
 * [David Young's overview](overview_david.md)
 * [SophieL's overview](overview_SophieL.md)
+* [Wendy Fish's overview](Overview_wuf.md)
```
Comment (rbwatson) [inline docs/index.md:28]: Added link to new overview in index.md.

**Review by rbwatson — APPROVED** [review]
LGTM.

**Pull request merged**